### DISCUSSION TITLE
这个应用会适配华为的HarmonyOS Next，纯血鸿蒙吗？

### DISCUSSION THREAD

**Discussion by HelloWorld-QY** [MetaCubeX/ClashMetaForAndroid#365] [General]
如题。

**Comment by sismeteromb** [comment]
https://github.com/233boy/v2ray/issues/1326
https://github.com/likuai2010/ClashMeta
https://github.com/likuai2010/auto-installer

**Reply by czxxxp** [reply]
谢谢🙏
